### ISSUE TITLE
Chat Commands requiring Admin

### ISSUE THREAD

**Issue by Zet0rz** [Zet0rz/nzombies#81]
This includes non-admin commands like /ready. 

You can work around it for now by doing one of two things:
- Make players super admins
- Run this code in the **server** console to ready all players up regardless of admin status:
  `lua_run for k,v in pairs(player.GetAll()) do v:ReadyUp() end`

**Comment by lemonylink** [comment]
We can just put this in an .lua file with the right code and on spawn it should Run when a player is in the server right? With the right code of course.

**Comment by Zet0rz** [comment]
@lemonylink No, this code isn't automatically run and will even create errors if you put it in a lua file.
`lua_run` is a console command that makes the following text be executed as lua code. This means you're essentially running the same code any gamemode or addon does, but in the console. You could theoretically code a full gamemode in a single `lua_run` statement if it didn't have a character limit.

Basically, this code loops through all players and runs player:ReadyUp() on them, which is what the server does when it registers a player (bugged to only do so on admins) say /ready. This means you need to put this in the console _when_ you want players to ready up, which is in-game, not in a lua file.

**Comment by lemonylink** [comment]
Ah right, I should of realised that. Completely Forgot lua_run was blocked.. Thank you. I'll just set it up to auto type it every 1-2 mins in the console then on a different machine. Your effort on this project is much appreciated.

**Comment by Zet0rz** [comment]
Fixed in commit af7dd5ea510163786c41dc183c30da6a33b29746
